calico_node/Makefile

```diff
@@ -1,6 +1,6 @@
 ###############################################################################
 # Versions:
-RELEASE_STREAM?=v2.3
+RELEASE_STREAM?=master
```

Comment:
> Update the RELEASE_STREAM variable at the top of calico_node/Makefile to the new major/minor version.

I think the weird bit is that RELEASE_STREAM is currently updated as part of the release process, so changing this to master doesn't really jive with that.

I think we need to revert this change and potentially change it in the near future, but only after thinking about impacts to our release process.

In the meantime, `make RELEASE_STREAM=master st` will work (though it's pretty annoying).